.changeset/hip-eels-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit': patch
+'lit-html': patch
+---
+
+export StaticValue type
```

Comment:
Nit. This becomes a line in the changelog so it would be nice if it's a bit more verbose like
```suggestion
`StaticValue` interface type is now exported.
```